### PR TITLE
CB-513 renaming cluster definition test related classes (TestDto/Test…

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RecommendationsTests.java [hunk 1]
```diff
@@ -66,14 +65,14 @@ private String getJsonFile() throws IOException {
 
     private void createClusterDefinition() throws Exception {
         given(CloudbreakClient.created());
-        given(ClusterDefinition.request().withName(VALID_CD_NAME).withDescription(CD_DESCRIPTION).withClusterDefinition(getClusterDefinitionFile()));
-        when(ClusterDefinition.post());
+//        given(ClusterDefinition.request().withName(VALID_CD_NAME).withDescription(CD_DESCRIPTION).withClusterDefinition(getClusterDefinitionFile()));
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RecommendationsTests.java]: what will be the fate of this test class? who will rewrite it with the new framework?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
     @BeforeSuite
     public void getAllClusterDefinitions() throws Exception {
         given(CloudbreakClient.created());
-        when(ClusterDefinition.getAll());
+//        when(ClusterDefinitionTestAction.getAll());
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java]: If this config class is no longer needed, then delete it completely. There is already too much garbage in the int tests.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RecommendationsTests.java (hunk 1 above)
Comment (topolyai5) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RecommendationsTests.java]: I don't know. I've created a task for it: https://jira.cloudera.com/browse/CB-615

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java (hunk 2 above)
Comment (topolyai5) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/CloudbreakClusterTestConfiguration.java]: After all of the old tests are rewritten we could delete dead codes

**Pull request merged**